chess-AI/self_play_dataset.py

```diff
@@ -0,0 +1,15 @@
+from torch.utils.data import Dataset
```

Comment:
If you're importing `torch`, just do `torch.utils.data.Dataset` below and get rid of this import.